GiniVision/Classes/CameraViewController.swift

```diff
@@ -283,7 +293,11 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     
     fileprivate func getVideoOrientation(forDeviceOrientation orientation:UIDeviceOrientation) -> AVCaptureVideoOrientation{
         if UIDevice.current.isIpad {
-            return orientationsMapping[UIDevice.current.orientation] ?? .portrait
+            if UIDevice.current.orientation.isFlat { // This only could happen on initialization
+                return interfaceOrientationsMapping[UIApplication.shared.statusBarOrientation] ?? .portrait
```

Comment:
Same here as before we should use only the statusBarOrientation, if there are no side-effects.